docs/architecture.md

```diff
@@ -2,65 +2,38 @@
 ### generator
-The generator project contains all logic to generate projects configured with the lirejarp project builder.
-
-### dbautoinit
-In this project, all database scripts are placed. Through flyway, these scripts are executed automatically in the DEV-mode. Two databases are supported: h2 and mysql. In system.properties of the tomee-configuration, you can configure, which database is used: flyway.database=h2 lirejarp project puilder with the additional subprojects.
-
-![Component Diagram LireJarp Project Builder](https://wp.starwit.de/ljprojectbuilder/wp-content/uploads/sites/7/2017/02/komponentendiagrammPS.png)
-
-## About implementing data access
+The generator project contains all logic to generate projects configured with the ProjectBuilder.
 
-You have a group of objects which are different in its structure but should implement the same basic behavior. You have a lots of data objects in your application for whom the standard CRUD-functionality should be provided? For this purpose, an AbstractDao is a common way to handle the implementation of the basic stuff. An AbstractEntity is created as base class for all entities. In the AbstractDao general CRUD functions are implemented using the AbstractEntity. E.g. http://sidaof.sourceforge.net/sidaof/ is a project, which implements an AbstractDao-class. Binding concrete entities to AbstractDao is done via generics.
-
-The architecture presented here goes further and uses this pattern through all layers of the entire architecture: business services (EJBs), RESTful webservices and junit-tests. Data objects extends the class “AbstractEntity”. Each kind of service (EJB, restful webservice, jUnit-testclass) implements the general functionality around the data objects in an abstract class. For each data object and each kind of service concrete service-implementations are added. To prevent class-casts, generics ensure the usage of concrete data object in the abstract part of the implementation.
-
-The following class-diagram provides an overview of the architecture. The description afterwards is given a better understanding of the diagram.
-
-![classdiagramm](https://wp.starwit.de/wp-content/uploads/2016/10/classDiagramm-1.png)
-
-### Entities for data access
-The lirejarp template uses JPA for data access. Hence, database table definitions and validation is defined in the entities.
-
-### Services to deal with entities and save data
-With the help of the generic approache, all entities can be saved and deleted using the same functionality. The implementation of the service class contains only specific behavior.
+Code generation via freemarker enables publishing of an entire running application via lirejarp project builder. The intent is to help to do the first step and using a standardized way to implement standard functionality. So, you will just generate once and change the code in the way you want. This pragmatic approach avoids a complex code generation.
```

Comment:
What is the differentiation between lirejarp, the projectbuilder and the lirejarp projectbuilder? I think this is not quite clear in this documentation and could be confusing for a potential reader